google-drive/1.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,48 @@
+= Google Drive Connector 1.0
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Google Drive (Google Drive Connector) gives Anypoint users the ability to automate business processes between Google Drive (Drive) and other enterprise applications, either on premises or in the cloud. Using the Google Drive connector, businesses can access and upload content and relevant information from and to Drive, programmatically.
+
+Drive gives enterprise users the ability to store and back up all their files, including documents, slides, images, audio and video, in the cloud. Drive integrates seamlessly with cloud-native apps that enable teams to collaborate effectively in real time, such as Google Docs, Sheets, and Slides.
+ 
+For compatibility information and fixed issues, see the Google Drive Connector Release Notes.
+
+== Prerequisites
+
+To use this connector, you should be familiar with:
+
+* The Google Drive API
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements and global elements in a Mule flow
+* Creating a Mule app using Anypoint Studio (Studio)
+
+Before creating an app, you must have:
+
+* Credentials to access Google Drive
+* Anypoint Studio version 4.2.1 or later
+
+== Common Use Cases for the Connectors
+
+Automate powerful business processes by integrating Google Drive with other IT tools and systems in your enterprise such as messaging, CRM, ITSM, and other enterprise systems. Possibilities include:
+
+* Improving sales operations 
++
+Automate contract approval processes so that when a sales contract is received in the Salesforce CRM system, it gets uploaded to Drive and shared with the deal desk or other teams for their review, automatically. 
+* Enhancing cross-team collaboration 
++
+Automatically notify all relevant teams and team members via Slack or Microsoft Teams and email, for review any time new content, for example, product specs, contracts, and legal addendums, is uploaded to a given folder in Drive.
+* Securing your enterprise content 
++
+Automatically upload content that is sent or uploaded via any other enterprise system, such as CRM, email, and marketing tools, to Drive folders that are configured with the appropriate security policies. This enables Drive to be the single source of truth for all of your content, which minimizes IP leaks.
```

Comment:
Is the content properly secured, or the Drive folder?

If the former, consider...:
```suggestion
Automatically upload properly secured content from other enterprise systems, such as CRM, email, and marketing tools, to Drive folders. This minimizes IP leaks by making Drive the single source of truth for all of your content.
```
If the latter, consider:
```suggestion
Automatically upload content Automatically upload content from other enterprise systems, such as CRM, email, and marketing tools, to properly secured Drive folders. This minimizes IP leaks by making Drive the single source of truth for all of your content.
```